string command
float32 fade_time
---
bool success
string message
